app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(comand: str) -> None:
+    names = comand.split(" ")
+    if names[1] == names[2]:
```

Comment:
u may do 1, 2, 3 = comand.split(" ") unstead using names[1] and ect.

Reply:
agree, you can unpack values into variables, and than use them it your code, but don't use `" "` inside of split(), because it is a default value